pkgs/development/python-modules/sphinxcontrib-confluencebuilder/default.nix

```
@@ -0,0 +1,39 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, docutils
+, sphinx
+, requests
+, jinja2
+}:
+
+buildPythonPackage rec {
+  pname = "sphinxcontrib-confluencebuilder";
+  version = "1.8.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-u+sjhj/2fu8fLGRb2zgnNI+y7wIIUYTMJhRekrdtMeU=";
+  };
+
+  propagatedBuildInputs = [
+    docutils
+    sphinx
+    requests
+    jinja2
+  ];
+
+  # Something about the way the unit tests are structured in this project is causing them to fail
```

I got tests running, but they have a circular dependency on Sphinx, which seems... messy. Per [this package](https://github.com/NixOS/nixpkgs/blob/180147d3dc9df04abc958e15417451096d95a414/pkgs/development/python-modules/sphinxcontrib-serializinghtml/default.nix#L17), we should do the same here.

```suggestion
  # Tests are disabled due to a circular dependency on Sphinx
```